quboo.properties

```diff
@@ -1,8 +1,8 @@
 category=Visualization/Reporting
 description=Provides integration with Quboo to use Gamification techniques to fix your legacy code.
 homepageUrl=https://quboo.io
-archivedVersions=
-publicVersions=1.0.3
+archivedVersions=1.0.3
```

Comment:
I can't comment on the relevant line (#11) so I'll comment here. Only one version can be compatible with `LATEST`, and right now you've got that for both versions. The range end for v1.0.3 needs to be updated. The convention is to set it at the version of SQ that was current at the time of the replacing release, so 7.8 at this point.

Reply:
Changed it  ✅